backend_new/src/controllers/user.controller.ts

```diff
@@ -63,4 +74,83 @@ export class UserController {
   ): Promise<User> {
     return this.userService.findOne(userId);
   }
+
+  @Put('forgot-password')
+  @ApiOperation({ summary: 'Forgot Password', operationId: 'forgotPassword' })
+  @ApiOkResponse({ type: SuccessDTO })
+  async forgotPassword(@Body() dto: EmailAndAppUrl): Promise<SuccessDTO> {
+    return await this.userService.forgotPassword(dto);
+  }
+
+  @Put(':id')
+  @ApiOperation({ summary: 'Update user', operationId: 'update' })
+  @ApiOkResponse({ type: User })
+  async update(@Body() dto: UserUpdate): Promise<User> {
+    return await this.userService.update(dto);
+  }
+
+  @Delete()
+  @ApiOperation({ summary: 'Delete user by id', operationId: 'delete' })
+  @ApiOkResponse({ type: SuccessDTO })
+  async delete(@Body() dto: IdDTO): Promise<SuccessDTO> {
+    return await this.userService.delete(dto.id);
+  }
+
+  @Post()
+  @ApiOperation({
+    summary: 'Creates a public only user',
+    operationId: 'create',
+  })
+  @ApiOkResponse({ type: User })
+  async create(
+    @Body() dto: UserCreate,
+    @Query() queryParams: UserCreateParams,
+  ): Promise<User> {
+    return await this.userService.create(
+      dto,
+      false,
+      queryParams.noWelcomeEmail !== true,
+    );
+  }
+
+  @Post('/invite')
+  @ApiOperation({ summary: 'Invite partner user', operationId: 'invite' })
+  @ApiOkResponse({ type: User })
+  async invite(@Body() dto: UserInvite): Promise<User> {
+    return await this.userService.create(dto, true);
+  }
+
+  @Post('resend-confirmation')
+  @ApiOperation({
+    summary: 'Resend public confirmation',
+    operationId: 'resendConfirmation',
+  })
+  @ApiOkResponse({ type: SuccessDTO })
+  async confirmation(@Body() dto: EmailAndAppUrl): Promise<SuccessDTO> {
+    return await this.userService.resendConfirmation(dto, false);
```

Comment:
I think you got this and the `resend-partner-confirmation` calls backwards. The `forPublic` flag is set to false here but should be true and vis-versa for `resend-partner-confirmation`

Reply:
Nice catch, got this updated!